ml-operator/README.md

```diff
@@ -19,10 +19,12 @@ vim .env
 # create a generic secret
 kubectl create secret generic ml-operator-secrets --from-env-file=.env
 
-# install ml-operator
+# install ml-operator from this directory
 helm upgrade --install ml-operator ./
-```
 
+# install ml-operator from the published mojaloop charts, overriding the secret name
+helm upgrade --install {release-name} mojaloop/ml-operator --set secret_name=ml-operator-secrets
```

Comment:
```suggestion
helm upgrade --install {release-name} mojaloop/ml-operator --set secret-name=ml-operator-secrets
```

Typo fix?

Reply:
I called it `secret-name` in the `values.yaml`, so I guess I should change it there as well?